src/libraries/System.Runtime.CompilerServices.VisualC/README.md

```diff
@@ -0,0 +1,17 @@
+# System.Runtime.CompilerServices
+Provides functionality for compiler writers who use managed code to specify attributes in metadata that affect the run-time behavior of the common language runtime.
```

Comment:
The directory is named System.Runtime.CompilerServices.**VisualC**, but the readme talks about System.Runtime.CompilerServices, is this correct? Most of the CompilerServices APIs are in System.Runtime, i think this readme should talk about C++/CLI specific APIs in `System.Runtime.CompilerServices.VisualC`.

Reply:
Yes, you are right, and I was aware of it was mostly for C++/CLI, on reason of adding these readmes is to let contributors know what types of community contributions we would accept. I made it to be `System.Runtime.CompilerServices` specific because there is no other package available, and this package was the closest and it is not independent package anymore.

 Though we might not want it this way, I will the comment open for further discussion

Reply:
For consistency, we should probably just call this S.R.CS.VisualC as it relates to the package. A note can be added here that the S.R.CS namespace has implementations mostly in the runtime.

Reply:
OK